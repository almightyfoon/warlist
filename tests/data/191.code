c1501b_-0n9a9EiPiPiPiPf6akazanaj9c___-0n7Cf6c0akanaj9be59b9b