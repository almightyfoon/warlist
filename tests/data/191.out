https://conflictchamber.com/#c1501b_-0n9a9EiPiPiPiPf6akazanaj9c___-0n7Cf6c0akanaj9be59b9b

Cygnar Army - 79 / 75 points
[Theme] Heavy Metal

!!! This army must contain 75 points or fewer

[Sloan 1] Captain Kara Sloan [+28]
 - Defender [16]
 - Hunter [10]
 - Hunter [10]
 - Hunter [10]
 - Hunter [10]
 - Hurricane [39]
 - Squire [0(5)]
Archduke Alain Runewood, Lord of Fharin [0(4)]
Captain Arlan Strangewayes [0(4)]
Journeyman Warcaster [4]
 - Firefly [8]

Cygnar Army - 79 / 75 points
[Theme] Heavy Metal

!!! This army must contain 75 points or fewer

[Stryker 1] Commander Coleman Stryker [+30]
 - Hurricane [39]
 - Stormwall [39]
 - Squire [0(5)]
Captain Arlan Strangewayes [0(4)]
Journeyman Warcaster [0(4)]
 - Charger [9]
Lieutenant Allison Jakes [4]
 - Charger [9]
 - Charger [9]
